package-lock.json

```diff
@@ -11211,7 +11359,7 @@
     },
     "is-data-descriptor": {
       "version": "0.1.4",
-      "resolved": "https://registry.npmjs.org/is-data-descriptor/-/is-data-descriptor-0.1.4.tgz",
+      "resolved": "http://registry.npmjs.org/is-data-descriptor/-/is-data-descriptor-0.1.4.tgz",
```

Comment:
why would you do this npm?

Reply:
So the npm maintainers have a proposed solution to this: https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285
Can you see if using "npm install --prefer-online" makes this go away? If so, we may wish to open an edX level ticket to fix it across some repos.

Reply:
hmmm yeah that didn't seem to fix it. I also tried clearing the cache and removing the modules before running it again just in case. :/
I'm just going to manually find & replace the http for now.